code/game/objects/items/storage/backpack.dm

```diff
@@ -535,6 +535,36 @@
 	icon_state = "intel_chestrig"
 	max_storage_space = 20
 
+/obj/item/storage/backpack/marine/satchel/intel/satchel
+	name = "\improper USCM expedition satchel"
+	desc = "A heavy-duty IMP based satchel, can quickly be accessed with only one hand. Reinforced with Kevlar so it doesn't rip. Usually issued to USCM intelligence officers."
+	icon_state = "marinesatch"
+	max_storage_space = 20
+
+/obj/item/storage/backpack/satchel/intelbrown
```

Comment:
you should probably make this obj/item/storage/backpack/marine/satchel/intel/satchel instead of the wrong path
also use snake case intel_brown

Reply:
although you dont really need to make intel/satchel you can just probably use satchel/intel_brown or something like that, but do fix it so the paths are the same